feedback.txt

```diff
@@ -0,0 +1,13 @@
+Lack of:
+  * i18n
+  * pagination
+  * delete task attachments
+  * separation between administration environment and signed-in user todo list
+
+I didn't understand why only Admin should be STI. Probably I misanderstood somthing, but to separate only Admin I presupposed that normal user are all Managers.
```

Comment:
Thats something we do with people doing the test inhouse here in Prague office to see how they communicate and ask for information (we had some bad experience, you know developers :) ) but you did it right, its just an excercise :)